### ISSUE TITLE
Feature request: blacklisting default properties

### ISSUE THREAD

**Issue by StevenVanbrielRosa** [mixpanel/mixpanel-react-native#266]
We care a lot about our users' privacy. Can you port this feature from the Javascript library to react native? 🙏

https://docs.mixpanel.com/docs/privacy/protecting-user-data#blacklisting-default-properties

**Comment by MykCib** [comment]
Yes, please. Maybe somebody found a workaround for react native to stop sending "Mixpanel properties" while still sending "Your properties"?

**Comment by Vishal-Lakkapathri** [comment]
@StevenVanbrielRosa , may be you can try registering them as super properties and give any value to it

```javascript
mixpanelInstance.registerSuperProperties({
   $manufacturer: "retracted",
   $city: 'retracted',
   $region: 'retracted',
   $carrier: 'retracted',
   $wifi: 'retracted'
})
```
